ansible/roles/influxdb/templates/influxdb.conf

```diff
@@ -0,0 +1,599 @@
+### Welcome to the InfluxDB configuration file.
+
+# The values in this file override the default values used by the system if
+# a config option is not specified. The commented out lines are the configuration
+# field and the default value used. Uncommenting a line and changing the value
+# will change the value used at runtime when the process is restarted.
+
+# Once every 24 hours InfluxDB will report usage data to usage.influxdata.com
+# The data includes a random ID, os, arch, version, the number of series and other
+# usage data. No data from user databases is ever transmitted.
+# Change this option to true to disable reporting.
+# reporting-disabled = false
+
+# Bind address to use for the RPC service for backup and restore.
+# bind-address = "127.0.0.1:8088"
+bind-address = "{{influxdb_host}}:{{influxdb_port}}"
+
+###
+### [meta]
+###
+### Controls the parameters for the Raft consensus group that stores metadata
+### about the InfluxDB cluster.
+###
+
+[meta]
+  # Where the metadata/raft database is stored
+  {% if influxdb_inmemory %}dir = "/dev/shm/influxdb/meta"{% else %}dir = "/var/lib/influxdb/meta"{% endif %}
+
+  enabled = true
```

Comment:
Let's have these three items occur, only when building the cluster, and not set them by default.